wolfcrypt/src/sp_int.c

```diff
@@ -12340,9 +12340,6 @@ int sp_read_unsigned_bin(sp_int* a, const byte* in, word32 inSz)
     return err;
 }
 
-#if (!defined(NO_DH) || defined(HAVE_ECC) || defined(WC_RSA_BLINDING) || \
```

Comment:
Is removal of this macro intentional? Assuming it is

Reply:
Fix for: ./configure --enable-sp-math-all --disable-dh --disable-ecc --enable-cryptonly --disable-harden
You mentioned this a while ago and I only saw it now.